common/reviews/api/heft.api.md

```diff
@@ -28,6 +29,20 @@ export type Build = IBuildActionData;
 
 // @public (undocumented)
 export class BuildHooks extends ActionHooksBase {
+    // (undocumented)
```

Comment:
This still seems odd to me:
```ts
export type Build = IBuildActionData;
```
Having multiple names for the same API is confusing.

Reply:
I'll change that in a follow-up.